### ISSUE TITLE
Error: WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name

### ISSUE THREAD

**Issue by DaveSkender** [GSA/touchpoints#1102]
We're running `pa11y` accessibility tests and getting hits on this specific rule on our modal form embedded as a tab.

```bash
Error: This textinput element does not have a name available to an accessibility API. 
Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name

<input type="text" name="fba_directive" id="fba_directive" style="display:none !important" 
  tabindex="-1" autocomplete="off">
```
cc @ttnguyen2

**Comment by ttnguyen2** [comment]
I re-tested the page after the touchpoint team fix. I don't see the error relating to #fba-directive anymore. This can be closed. However, I still get error relating to #star0. I reported a new issue for it.
